antismash/detection/hmm_detection/cluster_rules/loose.txt

```diff
@@ -29,3 +29,10 @@ RULE halogenated
     CUTOFF 5
     NEIGHBOURHOOD 10
     CONDITIONS Trp_halogenase
+
+RULE phosphonate-loose
```

Comment:
Without the `SUPERIORS` field set to the strict `phosphonate`, this rule will likely duplicate all or most of the stricter rule.

Reply:
Thanks for this - SUPERIORS has been set here. This also was a solution for "other" (catch all) rule that was calling hybrids for some phosphonate clusters

Reply:
I think that's the wrong fix for the `other` rule. As far as I can see what's causing the hybrid calls is the `fom1` profile, and that should just move to the phosphonates cluster rule instead. That profile is even called after one of the genes in your example cluster. :)

Reply:
Ok agreed thanks, I will just remove the fos1 model from the 'other' rule and remove the SUPERIORS flag. This is currently being hit with the updated phosphonate model so should not be missed.

Reply:
yeah, I guess that'd also be fine.

Reply:
Great, I just now pushed the changes and removed the fom1 model as it was not being used by any other rule